src/Stratis.Bitcoin/Controllers/Models/BanPeerViewModel.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+
+namespace Stratis.Bitcoin.Controllers.Models
+{
+    /// <summary>
+    /// Represents the model that will ban and disconnect a connected peer.
+    /// </summary>
+    public sealed class BanPeerViewModel
+    {
+        public BanPeerViewModel()
+        {
+            this.BanDurationSeconds = TimeSpan.FromDays(1).Seconds;
```

Comment:
Should we default this?

Reply:
`IPeerBanning.BanAndDisconnectPeer` requires a value to be passed, so defaulting it here in the constructor is OK?

Reply:
But the default should be taken from the settings

Reply:
Oh yes, my bad, I see it now in `Settings`, OK I will implement it now 👍